src/components/ebay-combobox/component.js

```diff
@@ -84,8 +90,14 @@ module.exports = {
     onInput(input) {
         input.autocomplete = input.autocomplete === 'list' ? 'list' : 'none';
         input.options = input.options || [];
-        this.lastValue = input.value;
-        this.state = { currentValue: input.value };
+        let inputId;
+        if (input['*']) {
```

Comment:
In marko 4, we should have `input.value` and `input.id` instead of `input['*']` attributes.

Reply:
What do you mean? I'm unfamiliar with the `['*']` syntax so I assume it meant something specific, since the referenced PR had `input['*'].id`. What's the difference between `input.id` and `input['*'].id`?

Reply:
So earlier marko v3 versions would put any additional parameters in the "*" object (basically anything not defined in `marko.json`). Markov4 basically removes that and puts all additional properties into the base object itself. 
IE taking `<ebay-component something=1/>`
Marko 3:
input["*"].something === 1
Marko 4:
input.something === 1

However when doing spread it always puts it in the base object.
```
$ var attrs = {something: 1}
<ebay-component ...attrs />
```

`input.something === 1` always

(Just for completeness sake and in case you use it in your project.... 
In `marko.json` we have a "@*" object which when set would set the additional parameters to whatever `targetProperty` you define. When setting the `targetProperty: null` then it will put the items into the base object which is what we have defined in our `marko.json`)

Reply:
OK, got it. So the resolution for these changes would be to not do the `input['*']` check, but just check for the property directly

Reply:
After changing the current code to 
```js
onInput(input) {
    input.autocomplete = input.autocomplete === 'list' ? 'list' : 'none';
    input.options = input.options || [];
    let inputId;
    if (input.id) {
        inputId = input.id;
    }
    this.lastValue = input.value;
    this.state = { currentValue: this.value, id: inputId };
}
```
I get a new failing test. My understanding is that this code should be functionally identical to existing code
```
1 failing

  1) src/components/ebay-combobox
       combobox
         renders with second item selected:
     Error: Found multiple elements with the role "option"

(If this is intentional, then use the `*AllBy*` variant of the query (like `queryAllByText`, `getAllByText`, or `findAllByText`)).
```

Reply:
Resolved with this recommendation: https://github.com/eBay/ebayui-core/pull/984#discussion_r357281806